dom/ajax/ajax-test.js

```diff
@@ -171,3 +171,18 @@ QUnit.asyncTest("crossDomain is true for relative requests", function(){
 		start();
 	});
 });
+
+if (__dirname !== '/') {
```

Comment:
In the canjs/canjs production tests, `__dirname` is set to `/`, which breaks all of these tests that are using `__dirname`. We could alternatively check `System.isEnv('production')` to skip the tests, but I'm not sure if that makes it any more obvious why this is being done.